src/Config/user_local.props

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <RevitVersionNumber Condition=" '$(RevitVersionNumber)' == '' ">2015</RevitVersionNumber>
+	<REVIT_VERSION>Revit_$(RevitVersionNumber)</REVIT_VERSION>
+    <DYNAMOAPI  Condition=" '$(DYNAMOAPI)' == '' ">$(SolutionDir)..\..\Dynamo\bin\$(Platform)\$(Configuration)</DYNAMOAPI>
+	<DynamoExternPath  Condition=" '$(DynamoExternPath)' == '' ">$(SolutionDir)..\..\Dynamo\extern</DynamoExternPath>
```

Comment:
We don't need this variable, we should use DYNAMOAPI. Think about developers using Dynamo installation to build, without cloning DynamoDS\Dynamo repo.